### DISCUSSION TITLE
Love this!

### DISCUSSION THREAD

**Discussion by ghost** [LeDragoX/Win-Debloat-Tools#150] [General]
Made my ram usage drop from using 8GB at idle all the way down to 3GB at idle. I have ran debloaters before and they have completely destroyed my system, but this one seems to work great. The GUI is amazing + it has the option to reverse any changes made by the debloater. Love this, excellent piece of software.

**Comment by LeDragoX** [comment]
I'm glad I could accomplish this and have a great review, thanks!
